packages/lab/src/progress/CircularProgress/CircularProgress.css

```diff
@@ -60,51 +48,52 @@
   display: inline-block;
 }
 
-.saltCircularProgress-static {
+.saltCircularProgress-determinate {
   transition: transform 0.3s cubic-bezier(0.4, 0, 0.2, 1) 0s;
 }
 
-.saltCircularProgress-indeterminate {
-  animation: circular-rotate 1.4s linear infinite;
-}
-
 .saltCircularProgress-svg {
+  height: calc(var(--salt-size-base) * 2);
+  width: calc(var(--salt-size-base) * 2);
   fill: var(--salt-accent-background);
   display: block;
 }
 
 .saltCircularProgress-circle {
+  r: var(--progress-circle-radius);
+  stroke-width: calc(var(--salt-size-detail) * 0.5);
```

Comment:
@pseys --salt-size-detail * 0.5 would be --salt-size-adornment now in the new token space, but it doesn't fit for this usage. should it be renamed? should another size be used?